.github/workflows/doc-build.yml

```diff
@@ -42,7 +43,10 @@ jobs:
 
     - name: install-pkgs
       run: |
-        sudo apt-get install -y ninja-build doxygen graphviz
+        sudo apt-get install -y ninja-build graphviz libclang1-9 libclang-cpp9
```

Comment:
curious.
What relation is there to `clang` in this commit ?

That is not clear at all from the commit message, and someone looking at git history in future to figure out why clang was added here will have a hard time to figure it out.

A few words please.

Reply:
Updated commit message, these are Doxygen runtime dependencies.